### PR TITLE
fix(google api autocomplete)

### PR DESCRIPTION
**Pull request by sandymash**
- fixed to fetch addressed even without commas
- remove IL comp restriction

### CODE REVIEW

**Review by ItayBenMoshe — DISMISSED** [review]

**Review by eladyashar — APPROVED** [review]

File: client/src/commons/LocationInputField/LocationInput.tsx [hunk 1]
```diff
@@ -73,48 +74,27 @@ const LocationInput = (props: LocationInputProps) => {
         _isMounted && setSelectedAddress(geoCodedAddress as GoogleApiPlace);
     };
 
+    const AutocompleteComponent =
+        <AutocompletedField
+            required={required}
```
Comment (eladyashar) [inline client/src/commons/LocationInputField/LocationInput.tsx:79]: require  attribute behaviour is conflicting our form submit design.
in chrome browser it is not possible to submit form while there is unfinished required fields, so the browser wont let you fire submit event (aka - save the data of the current tab and move to other tab) if there is required field. 
this validation checked in the schema instead.
if you pass false to the require prop there is no problem but if you pass yes you wont be able to move tabs while the field is empty (unwanted behaviour) i suggest to remove this prop

Comment (sandymash) [reply client/src/commons/LocationInputField/LocationInput.tsx]: sure, removed. thanks

File: client/src/commons/LocationInputField/LocationInput.tsx [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-import { TextField } from '@material-ui/core';
 import { Controller, Control } from 'react-hook-form';
-import { Autocomplete, AutocompleteRenderInputParams } from '@material-ui/lab';
 
 import useDBParser from 'Utils/vendor/useDBParsing';
 
 import useStyles from './LocationInputFieldStyles';
 import LocationOptionItem from './OptionItem/LocationOptionItem';
 import useGoogleApiAutocomplete from './useGoogleApiAutocomplete';
+import AutocompletedField from "../AutoCompletedField/AutocompletedField";
```
Comment (ItayBenMoshe) [inline client/src/commons/LocationInputField/LocationInput.tsx:9]: single quote

File: client/src/commons/LocationInputField/LocationInput.tsx [hunk 3]
```diff
@@ -33,7 +32,7 @@ const noOptionsMessage = 'הקלידו מיקום תיקני לחיפוש...';
 
 
 const LocationInput = (props: LocationInputProps) => {
-    const { control, name, selectedAddress,  setSelectedAddress } = props;
+    const { control, name, selectedAddress,  setSelectedAddress, required=false } = props;
```
Comment (ItayBenMoshe) [inline client/src/commons/LocationInputField/LocationInput.tsx:35]: space before and after '='

**Review by ItayBenMoshe — APPROVED** [review]

**Review by roymaniv — DISMISSED** [review]

**Review by MayaSasson — APPROVED** [review]

**Review by MayaSasson — APPROVED** [review]

**Review by roymaniv — APPROVED** [review]

**Pull request merged**